test/unit_tests/modules/util/data-set-spec.js

```diff
@@ -1,11 +1,52 @@
 'use strict';
 
 var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+var dataSet = require( BASE_JS_PATH + 'modules/util/data-set' ).dataSet;
 
 var chai = require( 'chai' );
 var expect = chai.expect;
-var dataSet = require( BASE_JS_PATH + 'modules/util/data-set' );
+var jsdom = require( 'mocha-jsdom' );
+var sinon = require( 'sinon' );
+var sandbox;
+var baseDom;
+
+var HTML_SNIPPET =
+  '<div data-test-value-a="testValueA"' +
+        'data-test-value-B="testValueB"' +
+        'data-testValue-C="testValueC"' +
+        'data-test-ValuE-D="testValueD"' +
+        'data-TEST-value-E="testValueE" >' +
```

Comment:
yeah, I was just testing the values to ensure they were all properly [CamelCased](https://github.com/cfpb/cfgov-refresh/blob/071a7c339954e8e4b2ee38cfcf56d8187bae0d90/cfgov/unprocessed/js/modules/util/data-set.js#L10.).